cartography/intel/aws/__init__.py

```diff
@@ -78,7 +79,8 @@ def start_aws_ingestion(neo4j_session, config):
         "UPDATE_TAG": config.update_tag,
     }
     try:
-        boto3_session = boto3.Session()
+        profile_name = os.getenv("AWS_PROFILE", default="default") or "default"
```

Comment:
This should be parsed in cli.py add added to the config.

Reply:
AWS_PROFILE is a [standard environment variable that boto3 uses](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#environment-variable-configuration).  This PR allows us to follow that standard.

Reply:
We are not letting boto3 parse the env. It is still cartography setting a specific profile_name. So we are bypassing boto3 profile picking logic.

Reply:
Yeah, that's correct.

Reply:
Hm, yeah I see your point, we can talk about this more and rethink.

Reply:
I wonder if we actually needed this code change - we might have been able to get the same behavior by just performing `AWS_PROFILE=myprofile cartography`.

Reply:
I believe it only uses the AWS_PROFILE variable if you don't explicitly set the profile_name, which cartography does.

Reply:
Just checked, `boto3.Session().profile_name` will be set to `AWS_PROFILE` if the environment variable is set, otherwise `default`.  More than happy to revert these changes as no longer necessary.